goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["metalkor-bomb-ag",                           "metalkor-bomb",              4,["NEB"], "levels/metal_head_nest/boss_room"],
+["metalkor-wings-ag",                          "metalkor-wings",             4,["NEB"], "levels/metal_head_nest/boss_room"],
+["metalkor-egg-ag",                            "metalkor-egg",               4,["NEB"], "levels/metal_head_nest/boss_room"],
+["rift-ring-in-game-ag",                       "rift-ring-in-game",          4,["NEB"], "levels/metal_head_nest/boss_room"],
+["nest-unbroken-rocks-ag",                     "nest-unbroken-rocks",        4,["NEB"], "levels/metal_head_nest/boss_room"],
+["metalkor-distort-ag",                        "metalkor-distort",           4,["NEB"], "levels/metal_head_nest/boss_room"],
+["metalkor-rays-ag",                           "metalkor-rays",              4,["NEB"], "levels/metal_head_nest/boss_room"],
+["nest-gun-elevator-ag",                       "nest-gun-elevator",          4,["NEB"], "levels/metal_head_nest/boss_room"],
+["metalkor-fma-spinner-ag",                    "metalkor-fma-spinner",       4,["NEB"], "levels/metal_head_nest/boss_room"],
+["nestb-tail-bound-ag",                        "nestb-tail-bound",           4,["NEB"], "levels/metal_head_nest/boss_room"],
+["rift-occlude-ag",                            "rift-occlude",               4,["NEB"], "levels/metal_head_nest/boss_room"],
+["nestb-vis",                                  "nestb-vis",                  4,["NEB"], "levels/metal_head_nest/boss_room"],
+["consiteb-part",                              "consiteb-part",              3,["COB"], "levels/construction_site"],
+["tpage-3032",                                 "tpage-3032",                 4,["COB"], "levels/construction_site"],
+["tpage-3033",                                 "tpage-3033",                 4,["COB"], "levels/construction_site"],
+["tpage-3142",                                 "tpage-3142",                 4,["COB"], "levels/construction_site"],
+["tpage-3180",                                 "tpage-3180",                 4,["COB"], "levels/construction_site"],
+["tpage-3239",                                 "tpage-3239",                 4,["COB"], "levels/construction_site"],
+["kor-transform-ag",                           "kor-transform",              4,["COB"], "levels/construction_site"],
+["kor-break-ag",                               "kor-break",                  4,["COB"], "levels/construction_site"],
+["vin-door-ctyinda-ag-COB",                    "vin-door-ctyinda",           4,["COB"], "levels/construction_site"],
+["vin-door-ctyinda-ag-CIA",                    "vin-door-ctyinda",           4,["CIA"], "levels/haven/industrial"],
+["kor-break-ray-ag",                           "kor-break-ray",              4,["COB"], "levels/construction_site"],
+["consiteb-vis",                               "consiteb-vis",               4,["COB"], "levels/construction_site"],
```

Comment:
`levels/consite/` is a bit less verbose. Also, it looks like `consitea` is for the construction site level while `consiteb` is for the version with Kor and the other characters and cutscene?